src/themes/dojo/slidePane.m.css

```diff
@@ -1,34 +1,91 @@
 @import './variables.css';
 
+.root,
+.root * {
+	box-sizing: border-box;
+}
+
 .underlayVisible {
 	background: var(--underlay-bg);
 }
 
-.content {
-	font: var(--base-font);
+.pane {
+	background: var(--dojo-white);
+	border: var(--border-thin) solid var(--border-color);
+	box-shadow: var(--drop-shadow);
```

Comment:
I used `--box-shadow-dimensions` for use with custom colors for inputs, but the shadow dimensions were different for those. We should probably think of better naming 😝